setup.py

```diff
@@ -18,6 +18,7 @@
           'colorama',
           'argparse',
           'configparser',
+          'BeautifulSoup',
```

Comment:
It seemed to work fine for me.

`(python3) ❯ pip install beautifulsoup`

`Collecting beautifulsoup
  Downloading BeautifulSoup-3.2.1.tar.gz
Building wheels for collected packages: beautifulsoup
  Running setup.py bdist_wheel for beautifulsoup ... done
  Stored in directory: /Users/lakshay/Library/Caches/pip/wheels/5e/be/6d/ed01d5d434a821557b674c9da976f60b1b93d9009447eb9d16
Successfully built beautifulsoup
Installing collected packages: beautifulsoup
Successfully installed beautifulsoup-3.2.1`